### PR TITLE
home page

### PR DESCRIPTION
**Pull request by Asam237**
https://user-images.githubusercontent.com/34966088/222944005-80649ffb-37a6-4ffe-9537-751fef9bc327.mp4

### CODE REVIEW

File: frontend/components/Pagination.vue [hunk 1]
```diff
@@ -373,6 +316,10 @@ export default {
       return this.$store.state.ratings.filtercompany
     },
   },
+  async created() {
+    await this.fetchRatings()
+    // console.log('number of page', this.numberPage)
```
Comment (rakici) [inline frontend/components/Pagination.vue:321]: Should we remove this `console.log` ?

File: frontend/components/Publish.vue [hunk 2]
```diff
@@ -0,0 +1,14 @@
+<template>
+    <main class="flex items-center justify-center">
+        <h1
+            class="font-bold md:text-4xl text-center text-blueDark py-24 sm:py-32 md:py-36 xl:py-40 sm:w-3/4 xl:w-1/2 text-xl xl:text-4xl px-4 xl:px-0">
+            Your job offer will be published as soon as your payment is approuved
```
Comment (rakici) [inline frontend/components/Publish.vue:5]:
```suggestion
            Your job offer will be published as soon as your payment is approved
```

File: frontend/layouts/app.vue [hunk 3]
```diff
@@ -1,27 +1,81 @@
...
+                v-for="(item, index) in navs"
+                :key="index"
+                :to="`${item.link}`"
+                :class="`h-7 lg:w-20 lg:h-16 py-2 flex justify-center items-center text-sm lg:text-base font-bold ${
+                  setID === item.id ? 'nav-select' : 'nav-noselect'
+                }`"
+                @click.native="setID = item.id"
+              >
+                {{ item.name }}
+              </NuxtLink> -->
```
Comment (rakici) [inline frontend/layouts/app.vue:21]: Why are we keeping this comment ?

File: frontend/pages/add_salary.vue [hunk 4]
```diff
@@ -2,47 +2,30 @@
...
-        <Title
-          title="Fill the form and submit your contribution"
-          font-size="header"
-          only-title="onlyTitle"
-        />
-        <p
-          class="text-center text-xs sm:text-base pt-2 md:pt-4 font-normal"
-          style="color: #2e2e2e; font-family: 'Inter', sans-serif"
-        >
+        <Title title="Fill the form and submit your contribution" font-size="header" only-title="onlyTitle" />
```
Comment (rakici) [inline frontend/pages/add_salary.vue:5]:
```suggestion
        <Title title="Fill in the form and submit your contribution" font-size="header" only-title="onlyTitle" />
```

**Review by rakici — APPROVED** [review]
It looks great overall, I left a few comments to fix some typos and remove useless comments.

**Comment by rakici** [comment]
We can't see which page currently has the focus.

Screenshot 2023-03-04 at 21 42 25

**Comment by Theryx** [comment]
Thanks, Asam for the work so far.
Please remove the image at the bottom of the home page
I see that the design is a little bit ahead of this implementation. Please check the Figma file to update the changes

**Pull request merged**